docs/reference/hacl/hash/blake2/blake2b.md

```diff
@@ -66,69 +80,44 @@ The following example shows how to use the one-shot API with the base implementa
 ````
 `````
 
+**Example**
+
 `````{tabs}
 ````{group-tab} 32
-```{doxygentypedef} Hacl_Streaming_Blake2_blake2b_32_state
-```
+There is no example for now.
```

Comment:
If we add something like this, let's add an issue that tracks adding the examples.

Reply:
I'll open some issues later. But I think we can also leave some open for now and mark them as "good first issue".

Reply:
https://github.com/cryspen/hacl-packages/issues/193